differ/differ.go

```diff
@@ -294,7 +302,183 @@ func evaluateFilterExpression(eventFilter string, thresholds EventThresholds, ev
 	return evaluator.Evaluate(eventFilter, values)
 }
 
-func processReportsByCluster(ruleContent types.RulesMap, storage Storage, clusters []types.ClusterEntry) {
+// ccx_rules_ocp.external.rules.cluster_wide_proxy_auth_check
+// ->
+// cluster_wide_proxy_auth_check
+func ruleIDToRuleName(ruleID types.RuleID) types.RuleName {
+	result := strings.TrimPrefix(string(ruleID), "ccx_rules_ocp.")
+	result = strings.TrimPrefix(result, "external.")
+	result = strings.TrimPrefix(result, "rules.")
+	result = strings.TrimPrefix(result, "bug_rules.")
+	result = strings.TrimPrefix(result, "ocs.")
+
+	return types.RuleName(result)
+}
```

Comment:
1) might be moved to https://github.com/RedHatInsights/insights-operator-utils/blob/master/generators/rule.go
2) why not to just get the substring after last `.`? (`string.LastIndex` or so)

Reply:
You are right, will modify according to 2).

Reply:
Done